packages/vue3/src/components/Card/Card.vue

```diff
@@ -0,0 +1,54 @@
+<script lang="ts" setup>
+import { useSlots } from 'vue';
+
+const slots = useSlots();
+</script>
+
+<template>
+  <div class="card">
+    <div v-if="slots.header" class="card-header">
+      <slot name="header" />
+    </div>
+    <div v-if="slots.body" class="card-body">
+      <slot name="body" />
+    </div>
+    <div v-else>
+      <slot />
+    </div>
+    <div v-if="slots.footer" class="card-footer">
+      <slot name="footer" />
+    </div>
+  </div>
+</template>
+
+<style scoped lang="scss">
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  overflow: hidden;
+  border-radius: 8px;
+  background-color: var(--base-white);
+  box-shadow: 0 0 16px 0 #1219261a;
```

Comment:
To keep things consistent it is better to utilize the existing variables, refer to the variable name in Figma